### PR TITLE
Float logo wrapper for better display of longer status page titles

### PR DESCRIPTION
**Pull request by jamesmacwhite**
# Description

On smaller viewports the status page logo and title display can better optimised, by having the logo wrapper as float, so the status page title is more neatly displayed. Essentially not trying to vertically align the title with the logo as viewport had less width for this to occur or have any real impact.

Currently without floating:

image

When applying a float:

image

## Type of change

- User Interface

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I ran ESLint and other linters for modified files
- [X] I have performed a self-review of my own code and test it
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [] My code needed automated testing. I have added them (this is optional task)

### CODE REVIEW

File: src/assets/app.scss [hunk 1]
```diff
@@ -14,6 +14,13 @@ h2 {
     font-size: 26px;
 }
 
+
+@media (max-width: 550px) {
```
Comment (Saibamen) [inline src/assets/app.scss:18]: Double enter
```suggestion
@media (max-width: 550px) {
```

**Review by Saibamen — CHANGES_REQUESTED** [review]

File: src/assets/app.scss (hunk 1 above)
Comment (jamesmacwhite) [reply src/assets/app.scss]: Thanks, forced pushed and fixed.

**Comment by jamesmacwhite** [comment]
@Saibamen Is this OK to be merged?

**Review by Saibamen — APPROVED** [review]

**Comment by Saibamen** [comment]
OK

**Comment by louislam** [comment]
For single line, it is no longer aligned to the middle.

image

**Comment by louislam** [comment]
I fixed it by using flexbox. Thanks for raising the issue.
https://github.com/louislam/uptime-kuma/pull/863/commits/c0c641998093f0690fd0b83206e73f2568993fd2

image

**Pull request closed without merging**